outputs.tf

```diff
@@ -1,7 +1,9 @@
 output "id" {
-  value = resource.null_resource.this.id
+  description = "ID to pass other modules in order to refer to this module as a dependency."
```

Comment:
"The module identifier. It may be used in depends_on of other modules to handle race conditions."